torchvision/prototype/datasets/__init__.py

```diff
@@ -7,7 +7,7 @@
         "Note that you cannot install it with `pip install torchdata`, since this is another package."
     ) from error
 
-from . import decoder, utils
+from . import decoder, utils, datapipes
```

Comment:
I've added a new module for this, because this datapipe probably shouldn't be an internal one since it is a useful building block if users want to construct their own dataset.